Orange/widgets/regression/owsgdregression.py

```diff
@@ -44,6 +45,11 @@ class OWSGDRegression(OWProvidesLearner, widget.OWWidget):
     want_main_area = False
     resizing_enabled = False
 
+    loss_function_options = ["Squared loss","Huber","Epsilon insensitive",
+                            "Squared Epsilon insensitive"]
+    penalty_options = ["Absolute norm (L1)","Euclidean norm (L2)",
+                       "Elastic Net (L1 and L2)"]
+    learning_rate_options = ["Inverse scaling","Constant"]
```

Comment:
With multiple values like that, I guess it's best to list the one per line, [like it was originally](https://github.com/biolab/orange3/pull/1107/files#diff-8694470c8acf1a953aef01d51dc21efeL69).
And since those are [class constants](https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming), please make the identifiers uppercase.